HelpAssistHer/help-assist-her
# Add modals

## Description
Add modals to drop down on form submit. A success modal drops down on successful submit, and an error modal should drop down on a failed submit. Modals can be closed manually and close with a CSS transition. 

Once down, modals can be closed by pressing the X on the top right of the modal, or by clicking anywhere in the form. 

## Test Plan
1. Click 'Verify Next Resource`
2. Submit Form
3. Scroll to top to see modal.
